src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinJvmTaskExecutor.kt

```diff
@@ -90,13 +88,6 @@ class KotlinJvmTaskExecutor @Inject internal constructor(
                 emptyList()
               }
             }
-            context.execute("javac") {
```

Comment:
It looks to me that we were compiling java twice, once here and once in https://sourcegraph.com/github.com/bazelbuild/rules_kotlin/-/blob/kotlin/internal/jvm/compile.bzl?L699  

I'm still testing this against our code base but so far it seems like it's safe to remove this -- didn't notice huge perf gains but we dont have that much java in our codebase